hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/mysql/MySQLCreateDataConnectionJdbcSqlConnectorTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.connector.jdbc.mysql;
+
+import com.hazelcast.jet.sql.impl.connector.jdbc.CreateDataConnectionJdbcSqlConnectorTest;
+import com.hazelcast.test.annotation.NightlyTest;
+import com.hazelcast.test.jdbc.MySQLDatabaseProvider;
+import org.junit.BeforeClass;
+import org.junit.experimental.categories.Category;
+
+@Category(NightlyTest.class)
+public class MySQLCreateDataConnectionJdbcSqlConnectorTest extends CreateDataConnectionJdbcSqlConnectorTest {
```

Comment:
Since this supposed to be integration tests would not be better to have more real scenario tested for that (than create data connection and try `SHOW RESOURCES FOR`) - connect to db, insert something and read it would be better I think, WDYT? (the same for `PostgresCreateDataConnectionJdbcSqlConnectorTest`)

Reply:
MappingJdbcSqlConnectorTest, and inhering tests for supported DBs already do what you have described. 

I wanted this test to be different. SHOW RESOURCES FOR is listing the INFORMATION_SCHEMA of the DB. If CREATE DATA CONNECTION fails, the INFORMATION_SCHEMA should fail too.  So this validates the connection.